en/products/relay/matrix.ftl

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/relay/#pricing
+
+plan-matrix-title = Pricing
+
+## PRICING MATRIX SECTION
+
+plan-matrix-offer-title = Choose a level of protection that’s right for you
+plan-matrix-offer-body = { -brand-name-firefox-relay } protects your identity and your inbox with unique email masks. Add on even more security with { -brand-name-relay } phone masking, and save { $savings } by adding { -brand-name-vpn } protection.
+
+## Top Row Labels
+
+# Variables:
+#   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+plan-matrix-heading-features = Features
+plan-matrix-heading-plan-free = Limited email protection
+plan-matrix-heading-plan-premium = Email protection
+plan-matrix-heading-plan-phones = Email & phone protection
+plan-matrix-heading-plan-bundle-2 = Add { -brand-name-vpn } protection
```

Comment:
```suggestion
plan-matrix-heading-plan-bundle-2 = Add VPN protection
```

This should have never been a brand in the first place.

Reply:
Good catch.

Reply:
I'll remove it from brands.ftl.

Reply:
@stephaniehobson - you might want to refrain from removing it completely. It seems like strings in the VPN project are using it.